cmd/dlt/accountroles/cmd.go

```diff
@@ -100,6 +100,14 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	var deleteClassic, deleteHostedCP bool
+	// Default flow deletes both classic and hosted-CP roles
+	if !args.classic && !args.hostedCP {
+		deleteClassic, deleteHostedCP = true, true
+	} else {
+		deleteClassic, deleteHostedCP = args.classic, args.hostedCP
+	}
+
```

Comment:
If we are defaulting to true, can we not set the default to true on init?

Reply:
We are defaulting to true only if the user didn't provide `--hosted-cp` and `--classic` flags.
Maybe I should change the condition here to check if the flags were provided instead.

Reply:
Exported to a new function.
I returned the values instead of multiple `if` statements for each combination of the flags, and found it more concise. Let me know your thoughts.